### ISSUE TITLE
KeyCap Footprint, JLCPCB Required Changes for BOM and Placement Not Saved on 20231129

### ISSUE THREAD

**Issue by ForrestErickson** [PubInv/moonrat#135]
@melanielaporte 

I have refreshed my fork and updated my local clone. I opened the schematic and examined the KeyCap properties.

The changes you made to the footprint on the key cap to resolve the JLCPCB order problem did not get saved.
The footprint still says: See JLCPCB assembly notes for footprint  
Where IIRC you ware changing this field to the manufacturers part number to clear that JLCPCB order problem

See:
![image](https://github.com/PubInv/moonrat/assets/5836181/04042c22-48f4-48fe-989c-04beeb4ae38e)

I think it is that when you were editing in the Symbol Fields Table you did not prese the Apply Save & Continue button before pressing OK to close the dialog.
![image](https://github.com/PubInv/moonrat/assets/5836181/16c42d06-eff2-4e39-9f87-018e55622f4b)

**Comment by melanielaporte** [comment]
What appears in my Kicad 
<img width="1216" alt="Screenshot 2023-11-30 at 9 40 14 AM" src="https://github.com/PubInv/moonrat/assets/107901386/161f4d35-cfcd-475b-9574-75c0956d4657">

**Comment by melanielaporte** [comment]
Uploaded JLCPCB files are located in [issue #132](https://github.com/PubInv/moonrat/issues/132)

**Comment by ForrestErickson** [comment]
20231201 I again refreshed my fork and clone.  I had to resolve a file conflict (the step file for the TS6-THT-SQ.step.
Then a bunch of files came in.
Now when I open the the change is there.
I can only conclude that I did not properly refresh my fork before not noticing the conflict. 

I have success!
![image](https://github.com/PubInv/moonrat/assets/5836181/2710b504-c2ad-4970-a096-d0063db22a12)

Closing.
